lib/auth/auth.go

```diff
@@ -778,9 +778,9 @@ func (a *Server) CheckU2FSignResponse(user string, response *u2f.SignResponse) e
 	return nil
 }
 
-// ExtendWebSession creates a new web session for a user based on a valid previous sessionID,
-// method is used to renew the web session for a user
-func (a *Server) ExtendWebSession(user string, prevSessionID string, identity tlsca.Identity) (services.WebSession, error) {
+// ExtendWebSession creates a new web session for a user based on a valid previous sessionID.
+// Additional roles are appended to intial roles if there is an approved access request.
+func (a *Server) ExtendWebSession(user, prevSessionID, accessRequestID string, identity tlsca.Identity) (services.WebSession, error) {
```

Comment:
> ExtendWebSession(user, prevSessionID, accessRequestID string, identity tlsca.Identity) 

UI calls this method every 10min. Is it expected that it needs to provide the accessRequestID value on each subsequent session renew request?

Reply:
discussed in slack call